doc/introduction/photonic_hardware.rst

```diff
@@ -46,39 +46,23 @@ will execute.
 
 Jobs can also be submitted using the **non-blocking** :meth:`eng.run_async() <.RemoteEngine.run_async>`
 method. Unlike :meth:`eng.run() <.RemoteEngine.run>`, which returns a :class:`~.Results` object once the computation is
-complete, this method instead returns a :class:`~.Job` object directly that can be queried
+complete, this method instead returns a `xcc.Job` object directly that can be queried
```

Comment:
It might be nice to set up intersphinx in the `conf.py` file 🙂 That way, you can still do
```
:class:`xcc.job`
```

and it should automatically resolve to the correct page in the XCC docs.

Similar to how we have set up intersphinx between the pennylane and qml sphinx.

This could be quite nice, since I imagine users might want to quickly see the available methods and attributes on an XCC job

Reply:
Nice! As long as this isn't too tedious to set up or maintain, I say we should go for it.

Reply:
Done! Thanks @josh146. 🥳